pybossa/jobs.py

```diff
@@ -1004,6 +1006,29 @@ def notify_task_progress(info, email_addr, queue='high'):
                 queue=queue)
     enqueue_job(job)
 
+def get_weekly_admin_report_jobs():
+    """Return email jobs with weekly report to admins"""
+    send_emails_date = current_app.config.get('WEEKLY_ADMIN_REPORTS').lower()
+    recipients = current_app.config.get('WEEKLY_ADMIN_REPORTS_EMAIL')
+    today = datetime.today().strftime('%A').lower()
+    timeout = current_app.config.get('TIMEOUT')
+    if recipients and today == send_emails_date:
+        info = dict(timestamp=datetime.datetime.now().isoformat(),
```

Comment:
This should be datetime.now().isoformat() as import is "from datetime import datetime"